torch/csrc/jit/import.h

```diff
@@ -10,12 +10,17 @@ namespace jit {
 
 using ModuleLookup = std::function<std::shared_ptr<script::Module>(
     const std::vector<std::string>&)>;
+using DeviceLookup = std::function<std::string(at::Tensor&, const std::string&)>;
```

Comment:
should it receive Storage here to be consistent with torch.serialization API?

also, if we return only the device  string - maybe it doesn't make sense to pass in the storage at all?

@zdevito  - what do you think?

Reply:
Receiving Storage makes more sense. I will turn to it.
Maybe we can fully align with python API (returning a storage instead of device string). The only concern is that we may lost the control of copying/casting storage.